Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseAutonomous.java

```diff
@@ -223,4 +224,25 @@ public void onError(int errorCode) {}
             return tagOfInterest.id;
         }
     }
+
+    /**
+     *EXPERIMENTAL
```

Comment:
Is this still experimental? If not, remove this.

Reply:
Removed

Reply:
Why did this come back? Make sure you pull the latest changes before adding more changes on top.